### ISSUE TITLE
django.db.migrations error after restart

### ISSUE THREAD

**Issue by vavalomi** [WeblateOrg/weblate#2204]
### Steps to reproduce

1. Had a docker-based empty install working
2. Added new project/components, saw all the data populated
3. Stopped the container (removed debug setting, modified smtp settings...)
4. Trying to restart and getting this error

Here's what I get:

```
Attaching to weblate_cache_1, weblate_weblate_1
^[[36mcache_1    |^[[0m 1:C 30 Aug 18:42:02.246 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
^[[36mcache_1    |^[[0m 1:C 30 Aug 18:42:02.246 # Redis version=4.0.11, bits=64, commit=00000000, modified=0, pid=1, just started
^[[36mcache_1    |^[[0m 1:C 30 Aug 18:42:02.246 # Configuration loaded
^[[36mcache_1    |^[[0m 1:M 30 Aug 18:42:02.247 # You requested maxclients of 10000 requiring at least 10032 max file descriptors.
^[[36mcache_1    |^[[0m 1:M 30 Aug 18:42:02.247 # Server can't set maximum open files to 10032 because of OS error: Operation not permitted.
^[[36mcache_1    |^[[0m 1:M 30 Aug 18:42:02.247 # Current maximum open files is 4096. maxclients has been reduced to 4064 to compensate for low ulimit. If you need higher maxclients increase 'ulimit -n'.
^[[36mcache_1    |^[[0m 1:M 30 Aug 18:42:02.249 * Running mode=standalone, port=6379.
^[[36mcache_1    |^[[0m 1:M 30 Aug 18:42:02.249 # WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
^[[36mcache_1    |^[[0m 1:M 30 Aug 18:42:02.249 # Server initialized
^[[36mcache_1    |^[[0m 1:M 30 Aug 18:42:02.250 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.$
^[[36mcache_1    |^[[0m 1:M 30 Aug 18:42:02.250 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue$
^[[36mcache_1    |^[[0m 1:M 30 Aug 18:42:02.300 * DB loaded from append only file: 0.049 seconds
^[[36mcache_1    |^[[0m 1:M 30 Aug 18:42:02.300 * Ready to accept connections
^[[33mweblate_1  |^[[0m Postgres is up
^[[33mweblate_1  |^[[0m Traceback (most recent call last):
^[[33mweblate_1  |^[[0m   File "/usr/local/bin/weblate", line 11, in <module>
^[[33mweblate_1  |^[[0m     sys.exit(main())
^[[33mweblate_1  |^[[0m   File "/usr/local/lib/python3.5/dist-packages/weblate/runner.py", line 33, in main
^[[33mweblate_1  |^[[0m     execute_from_command_line(argv)
^[[33mweblate_1  |^[[0m   File "/usr/local/lib/python3.5/dist-packages/django/core/management/__init__.py", line 371, in execute_from_command_line
^[[33mweblate_1  |^[[0m     utility.execute()
^[[33mweblate_1  |^[[0m   File "/usr/local/lib/python3.5/dist-packages/django/core/management/__init__.py", line 365, in execute
^[[33mweblate_1  |^[[0m     self.fetch_command(subcommand).run_from_argv(self.argv)
^[[33mweblate_1  |^[[0m   File "/usr/local/lib/python3.5/dist-packages/django/core/management/base.py", line 288, in run_from_argv
^[[33mweblate_1  |^[[0m     self.execute(*args, **cmd_options)
^[[33mweblate_1  |^[[0m   File "/usr/local/lib/python3.5/dist-packages/django/core/management/base.py", line 335, in execute
^[[33mweblate_1  |^[[0m     output = self.handle(*args, **options)
^[[33mweblate_1  |^[[0m   File "/usr/local/lib/python3.5/dist-packages/django/core/management/commands/migrate.py", line 82, in handle
^[[33mweblate_1  |^[[0m     executor.loader.check_consistent_history(connection)
^[[33mweblate_1  |^[[0m   File "/usr/local/lib/python3.5/dist-packages/django/db/migrations/loader.py", line 292, in check_consistent_history
^[[33mweblate_1  |^[[0m     connection.alias,
^[[33mweblate_1  |^[[0m django.db.migrations.exceptions.InconsistentMigrationHistory: Migration trans.0099_remove_project_enable_acl is applied before its dependency permissions.0007_auto_20170510_1036 on d$
^[[33mweblate_weblate_1 exited with code 1
```

Since I can't get container running I'm unable to execute --list-versions but 'trust' me, this is the latest version, have pulled multiple times today...

**Comment by nijel** [comment]
It seems the database got into inconsistent state. Did you stop the database container properly or was it killed?

**Comment by vavalomi** [comment]
used docker-compose stop...

**Comment by nijel** [comment]
Okay, it might be something completely different. Didn't you happen to run older container after running 3.1.1? It actually seems you are loading 3.0 (or older) migrations code, while the database is migrated to 3.1.

**Comment by vavalomi** [comment]
I don't think that I'd even have older versions... only started playing with weblate this month, on only with docker image, haven't downloaded the source or anything...
This was literally the fist time I populated the data and restarted. I can simply clean the database and redo the work but wanted to make sure I understand if I did something wrong so that I don't end up in the same situation once more history is accumulated in production ... (of course, regular backups are a separate question)

**Comment by nijel** [comment]
Okay, I probaly found it out - for some reason Docker hub didn't pickup 3.1.1-5 as latest, but rather 3.0.1-7. I've just pushed 3.1.1-5 and that should hopefully fix latest tag. Once that is done please upgrade.

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).
